Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqTable.Tests.ps1

```diff
@@ -1,56 +1,117 @@
 ﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
 $project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
 
+$testDataset = "pshell_testing"
+
 Describe "Get-GcbqTable" {
 
-    #TODO(ahandley): Include setup and teardown code in these tests as New- and Remove- are developed.
+    $d = New-GcbqDataset $testDataset
```

Comment:
Also, we should use something meaningful for variable name. For example $dataset instead of $d.